[SUBS-1024] Handling POST request coming from file upload server (tusd)

ke4 opened the pull request.

fpenim commented:
There's an issue with you tests, it's complaining with `org.mockito.exceptions.misusing.InvalidUseOfMatchersException`. Otherwise everything looks ok 👌

ke4 commented:
I fixed my tests. I used an argument matcher in the `TokenHandlerServiceTest.java` what I supposed to use in a mock context. I replaced it to another matcher and it works in Travis as well.
The strange thing that the previously used matcher worked on my machine without any failure message.
